how-to-retrieve-cluster-yaml-from-custom-cluster/cluster-yaml-recovery.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+if [[ ! -f kube_config_cluster.yml ]]
+then
+  echo "The kube_config_cluster.yml is missing"
+  exit 1
+fi
+
+echo "Checking that jq is installed"
+if ! jq -V
+then
+  echo "Please download jq from https://github.com/stedolan/jq/releases/tag/jq-1.6 and install it"
+  exti 1
+fi
+
+echo "Checking that yq v3.x is installed"
+if ! `yq -V | grep 'yq version 3'`
+then
+  echo "Please download yq v3.x from https://github.com/mikefarah/yq/releases/tag/3.4.1 and install it"
+  exit 1
+fi
+
+echo "Building cluster.yml..."
+echo "Working on Nodes..."
+echo 'nodes:' > cluster.yml
```

Comment:
Add a guardrail to exit if `cluster.yml` exists.

Reply:
Fixed